src/blop/integrations/ax/helpers.py

```diff
@@ -78,7 +108,7 @@ def evaluate(parameterization: dict[str, float] | dict[str, list[float]]) -> dic
         uid = RE(plan_function(readables, *unpacked))
 
         # Fetch the data
-        results_df = db[uid][0].table(fill=True)
+        results_df = convert_to_dictonary(db[uid])
```

Comment:
Same thing here, use `data_access.get_data()`
```suggestion
        results = data_access.get_data(uid)
```